src/models/scatter.js

```diff
@@ -189,13 +213,19 @@ nv.models.scatter = function() {
                         [width + 10,-10]
                     ]);
 
-                    var voronoi = d3.geom.voronoi(vertices).map(function(d, i) {
-                        return {
-                            'data': bounds.clip(d),
-                            'series': vertices[i][2],
-                            'point': vertices[i][3]
-                        }
-                    });
+                    var voronoi = d3.geom.voronoi(vertices)
+                        .filter(function(d, i) {
+                            // Filter out dummy points
+                            return vertices[i][4] !== null;
+                        })
+                        .map(function(d, i) {
+                            return {
+                                'data': bounds.clip(d),
+                                'series': vertices[i][2],
+                                'point': vertices[i][3],
+                                'sibs': (vertices[i].length >= 6) ? vertices[i][5] : null
```

Comment:
There's heavy usage of magic numbers, which I realize has always been in the code.  But would be nice if it was put in variable